### DISCUSSION TITLE
Name the Shortcuts sites and increase the number of shortcuts allowed

### DISCUSSION THREAD

**Discussion by Nermeen3** [dailydotdev/daily#1231] [Feature Requests]
Hey Team,

I truly enjoy the daily dev homepage and appreciate all the efforts that everyone does.

Though, I have a minor suggestion:

I currently use brave browser and the homepage has a fantastic customizable shortcuts section where it allows you to:
1. have two pages where each page can contain 12 shortcut cards
2. Each card can have a custom name (daily dev doesn't have it yet)
3. The cards can be also re-ordered which is great

here's an image example from brave homepage that I have customized:

![image](https://github.com/dailydotdev/daily/assets/63965785/8b318721-4b62-4c02-9d49-e2752878339c)

And here's an image from the daily dev shortcuts section for comparison

![image](https://github.com/dailydotdev/daily/assets/63965785/a2b3bc04-8c3f-40d9-b11d-437bae6113a1)

I think improving the shortcuts section similar to the points above can greatly benefit the users who already had custom shortcuts to pin their most important sites but lost this minor customization after enabling this one of a kind homepage!

**Comment by idoshamun** [comment]
Hi @Nermeen3,

Thanks for the suggestion. Just to be clear do you expect to have more shortcuts or the name support (or both of course 😬 )?

**Comment by Nermeen3** [comment]
Thank you for the quick response, idoshamun! (Apologies, I didn't receive a notification and didn't notice the reply earlier.)

And yes, having both will be a chef's kiss <3
